plugins/essif-lab_contactform7/wordpress-plugin.php

```diff
@@ -17,34 +9,38 @@
  * Text Domain: essif-lab_contactform7
  */
 
-// If this file is called directly, abort.
-if ( ! defined( 'WPINC' ) ) {
+if ( ! defined( 'WPINC' ) || !defined('ABSPATH') ) {
 	die;
 }
 
-define( 'ESSIF_LAB_CONTACTFORM7_VERSION', '1.0.0' );
+$classAutoloader = __DIR__.'/vendor/autoload.php';
+if (file_exists($classAutoloader)) {
+    require_once($classAutoloader);
+}
+
+use TNO\ContactForm7\Application\Essif_Lab_contactform7_Activator;
+use TNO\ContactForm7\Application\Essif_Lab_contactform7_Deactivator;
+use TNO\ContactForm7\Application\Essif_Lab_contactform7;
 
 function activate_essif_lab_contactform7() {
-	require_once plugin_dir_path( __FILE__ ) . 'Application/class-essif-lab_contactform7-activator.php';
+	require_once plugin_dir_path( __FILE__ ) . 'src/Application/activator.php';
```

Comment:
`require_once` is hier ook niet nodig. De `require_once` van de autloader op regel 18 doet dit als het goed al (als het niet werkt moet je waarschijnlijk `composer dumpautoload` runnen).